qiskit/circuit/quantumcircuit.py

```diff
@@ -4234,14 +4238,14 @@ def mcx(
         num_ctrl_qubits = len(control_qubits)
 
         available_implementations = {
-            "noancilla": MCXGrayCode(num_ctrl_qubits),
-            "recursion": MCXRecursive(num_ctrl_qubits),
-            "v-chain": MCXVChain(num_ctrl_qubits, False),
-            "v-chain-dirty": MCXVChain(num_ctrl_qubits, dirty_ancillas=True),
+            "noancilla": MCXGrayCode(num_ctrl_qubits,ctrl_state=ctrl_state),
```

Comment:
I think there should be a space after the comma (CI checks seem to be stuck now).